### PR TITLE
Feature/favorite

### PR DESCRIPTION
**Pull request by nmemarcoding**
Add favorite icons whenever there is a favorite meal or drink. The icon will display favorite meals and drinks when clicked.

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -479,4 +479,169 @@ function mealStar() {
...
+            <div class="card-image has-text-centered px-6">
+            <img src="${recipeImage}" alt="${recipeTitle}">
+            <div class="top-right is-size-1-mobile"><i class="far fa-star meal-star" id="meal-star" data-title="${recipeTitle}"></i></div>
+            </div>
+            <div class="card-content">
+                <p class="title is-size-5">${recipeTitle}</p>
+            </div>
+            <footer class="card-footer">
+                <p class="card-footer-item">
+                    <button class="button is-link is-light has-background-white recipe-button" data-index="${data.meals[i].idMeal}">View Recipe</button>
```
Comment (hjohnsick) [inline assets/js/script.js]: Remove the, is-link is-light has-background-white, from the class so it has the same styling as the buttons on the other cards.

File: assets/js/script.js [hunk 2]
```diff
@@ -479,4 +479,169 @@ function mealStar() {
...
+            <div class="card-image has-text-centered px-6">
+                <img src="${recipeImage}" alt="${recipeTitle}">
+                <div class="top-right is-size-1-mobile"><i class="far fa-star drink-star" id="drink-star" data-title="${recipeTitle}"></i></div>
+            </div>
+            <div class="card-content">
+                <p class="title is-size-5">${recipeTitle}</p>
+            </div>
+            <footer class="card-footer">
+            <p class="card-footer-item">
+            <button class="button is-link is-light has-background-white drink-button" data-index="${data.drinks[i].idDrink}">View Recipe</button>
```
Comment (hjohnsick) [inline assets/js/script.js]: Remove the, is-link is-light has-background-white, from the class so it has the same styling as the buttons on the other cards.

**Review by hjohnsick — CHANGES_REQUESTED** [review]
Great work!  Just update the classes on the view recipe buttons so they have the same style as the other cards.

File: assets/js/script.js (hunk 1 above)
Comment (nmemarcoding) [reply assets/js/script.js]: fixed

File: assets/js/script.js (hunk 2 above)
Comment (nmemarcoding) [reply assets/js/script.js]: fixed

**Review by hjohnsick — APPROVED** [review]

**Pull request merged**